File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        for (int i : num_list) {
            int a = i;
            while (a > 1) {
                if (a % 2 == 0) {
                    a /= 2;
                } else {
                    a = (a - 1) / 2;
                }
                answer++;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.kt
class Solution {
    fun solution(num_list: IntArray): Int {
        var answer: Int = 0
        for (i in num_list) {
            var a = i
            while (a > 1) {
                if (a % 2 == 0) {
                    a /= 2
                } else {
                    a = (a - 1) / 2
                }
                answer++
            }
        }
        return answer
    }
}